disperser/apiserver/payment_test.go

```diff
@@ -0,0 +1,150 @@
+package apiserver_test
+
+import (
+	"context"
+	"crypto/rand"
+	"math/big"
+	"net"
+	"testing"
+	"time"
+
+	"github.com/Layr-Labs/eigenda/core/auth"
+	"github.com/Layr-Labs/eigenda/core/meterer"
+	"github.com/Layr-Labs/eigenda/core/mock"
+	"github.com/Layr-Labs/eigenda/encoding/utils/codec"
+
+	pbcommon "github.com/Layr-Labs/eigenda/api/grpc/common"
+	pb "github.com/Layr-Labs/eigenda/api/grpc/disperser"
+	"github.com/Layr-Labs/eigenda/core"
+	"github.com/stretchr/testify/assert"
+	tmock "github.com/stretchr/testify/mock"
+	"google.golang.org/grpc/peer"
+)
+
+func TestDispersePaidBlob(t *testing.T) {
+
+	transactor := &mock.MockWriter{}
+	transactor.On("GetCurrentBlockNumber").Return(uint32(100), nil)
+	transactor.On("GetQuorumCount").Return(uint8(2), nil)
+	quorumParams := []core.SecurityParam{
+		{QuorumID: 0, AdversaryThreshold: 80, ConfirmationThreshold: 100},
+		{QuorumID: 1, AdversaryThreshold: 80, ConfirmationThreshold: 100},
+	}
+	transactor.On("GetQuorumSecurityParams", tmock.Anything).Return(quorumParams, nil)
+	transactor.On("GetRequiredQuorumNumbers", tmock.Anything).Return([]uint8{0, 1}, nil)
+
+	quorums := []uint32{0, 1}
+
+	dispersalServer := newTestServer(transactor, t.Name())
+
+	data := make([]byte, 1024)
+	_, err := rand.Read(data)
+	assert.NoError(t, err)
+
+	data = codec.ConvertByPaddingEmptyByte(data)
+
+	p := &peer.Peer{
+		Addr: &net.TCPAddr{
+			IP:   net.ParseIP("0.0.0.0"),
+			Port: 51001,
+		},
+	}
+	ctx := peer.NewContext(context.Background(), p)
+
+	transactor.On("GetRequiredQuorumNumbers", tmock.Anything).Return([]uint8{0, 1}, nil).Twice()
+
+	pk := "0x0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdeb"
+	signer := auth.NewPaymentSigner(pk)
+
+	dataLength := len(data)
```

Comment:
Similarly to the `server.go` comment, I believe we should be using the number of symbols, here, correct?